styles/_icomoon.scss

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: 'icomoon';
+  src:  url('../icomoon/icomoon.woff?38a4jg') format('woff');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'icomoon' !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-search:before {
+  content: "\e912";
+}
+.icon-calendar:before {
+  content: "\e900";
+}
+.icon-check:before {
+  content: "\e901";
+}
+.icon-chevron-down:before {
+  content: "\e902";
+}
+.icon-chevron-left:before {
+  content: "\e903";
+}
+.icon-chevron-right:before {
+  content: "\e904";
+}
+.icon-fb:before {
+  content: "\e905";
+}
+.icon-footer_email:before {
+  content: "\e906";
+}
+.icon-footer_phone:before {
+  content: "\e907";
+}
+.icon-footer_pin:before {
+  content: "\e908";
+}
+.icon-gift:before {
+  content: "\e909";
+}
+.icon-home:before {
+  content: "\e90a";
+}
+.icon-inst:before {
+  content: "\e90b";
+}
+.icon-map-pin:before {
+  content: "\e90c";
+}
+.icon-phone-call:before {
+  content: "\e90d";
+}
+.icon-shopping-cart:before {
+  content: "\e90e";
+}
+.icon-star:before {
+  content: "\e90f";
+}
+.icon-user:before {
+  content: "\e910";
+}
+.icon-yt:before {
```

Comment:
`yt`? what does it mean? Could you please rename this icon?

Reply:
It means youtube)) just short